configs/akeneo-api.json

```diff
@@ -1,15 +1,19 @@
 {
   "index_name": "akeneo-api",
   "start_urls": [
-    "https://api.akeneo.com/documentation.html"
+    "https://api.akeneo.com/getting-started.html",
+    "https://api.akeneo.com/getting-started-admin.html",
+    "https://api.akeneo.com/getting-started-admin.html",
+    "https://api.akeneo.com/documentation/",
+    "https://api-staging.akeneo.com/api-reference-index.html"
   ],
   "selectors": {
-    "lvl0": "#includedContent h1",
-    "lvl1": "#includedContent h2",
-    "lvl2": "#includedContent h3",
-    "lvl3": "#includedContent h4",
-    "lvl4": "#includedContent h5",
-    "text": "#includedContent p, #includedContent li"
+    "lvl0": "h1",
```

Comment:
Most of the time it's a bad idea to select every h* in the page.
If you have a top menu or footer, it will also select those. 
Don't you have a content class or something similar?